config/routes.rb

```diff
@@ -6,10 +6,11 @@
   resources :users, only: [:index, :show]
 
   # Routes for pokemons
-  resources :pokemons, only: [:index, :show, :new, :create] do
+  resources :pokemons, only: [:index, :show, :new, :create, :edit, :update] do
     # Nested route on pokemons for bookings
-    resources :bookings, only: [:show, :new, :create]
+    resources :bookings, only: [:show, :new]
   end
 
+  resources :bookings, only: [:index, :create, :edit, :update]
```

Comment:
I'm afraid that the unested :create will create booking disconnected from the attached pokemon. 
Did you try it locally and confirm that you can create new bookings this way and still have them tied with pokemon?